Makefile

```diff
@@ -35,3 +35,15 @@ $(call add-bindata,etcd,./bindata/etcd/...,bindata,etcd_assets,pkg/operator/etcd
 # See vendor/github.com/openshift/build-machinery-go/scripts/run-telepresence.sh for usage and configuration details
 export TP_DEPLOYMENT_YAML ?=./manifests/0000_12_etcd-operator_06_deployment.yaml
 export TP_CMD_PATH ?=./cmd/cluster-etcd-operator
+
+# This was copied from https://github.com/openshift/machine-config-operator
+# Run tests
+test: test-unit test-e2e
+
+# Unit tests only (no active cluster required)
+test-unit:
+	CGO_ENABLED=0 go test -tags=$(GOTAGS) -count=1 -v ./cmd/... ./pkg/... 
+
+# This was copied from https://github.com/openshift/cluster-image-registry-operator
+test-e2e:
```

Comment:
for e2e can we align with KAS?

https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/Makefile#L70

Also we already have `test-unit` right? 

https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-release-4.6.yaml#L51

Reply:
`test-unit` was failing with e2e added. See previous test outputs:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-etcd-operator/394/pull-ci-openshift-cluster-etcd-operator-master-unit/1285873347153891328